src/blob_gc_job_test.cc

```diff
@@ -196,6 +210,76 @@ class BlobGCJobTest : public testing::Test {
     DestroyDB();
   }
 
+  void TestGCLimiter() {
+    class TestLimiter : public RateLimiter {
+     public:
+      TestLimiter(RateLimiter::Mode mode)
+          : RateLimiter(mode), read(false), write(false) {}
+
+      virtual size_t RequestToken(size_t bytes, size_t alignment,
+                                  Env::IOPriority io_priority,
+                                  Statistics* stats,
+                                  RateLimiter::OpType op_type) {
+        if (IsRateLimited(op_type)) {
+          if (op_type == RateLimiter::OpType::kRead) {
+            read = true;
+          } else {
+            write = true;
+          }
+        }
+        return bytes;
+      }
+
+      virtual void SetBytesPerSecond(int64_t bytes_per_second) {}
+
+      virtual int64_t GetSingleBurstBytes() const { return 0; }
+
+      virtual int64_t GetTotalBytesThrough(
+          const Env::IOPriority pri = Env::IO_TOTAL) const {
+        return 0;
+      }
+
+      virtual int64_t GetTotalRequests(
+          const Env::IOPriority pri = Env::IO_TOTAL) const {
+        return 0;
+      }
+
+      virtual int64_t GetBytesPerSecond() const { return 0; }
+
+      bool ReadRequested() { return read; }
+
+      bool WriteRequested() { return write; }
+
+     private:
+      bool read;
```

Comment:
need to reset both of these flags to false after flush, since flush also make use of rate limiter.